workflows/sock-shop-demo/usingPromProbe/workflow_cron.yaml

```diff
@@ -0,0 +1,689 @@
+            "kubectl apply -f /tmp/pod-memory-hog.yaml -n {{workflow.parameters.adminModeNamespace}}",
+          ]
+
+    - name: pod-cpu-hog
+      inputs:
+        artifacts:
+          - name: pod-cpu-hog
+            path: /tmp/chaosengine.yaml
+            raw:
+              data: |
+                apiVersion: litmuschaos.io/v1alpha1
+                kind: ChaosEngine
+                metadata:
+                  name: pod-cpu-hog-chaos
+                  namespace: {{workflow.parameters.adminModeNamespace}}
+                spec:
+                  appinfo:
+                    appns: 'sock-shop'
+                    applabel: 'name=carts'
+                    appkind: 'deployment'
+                  jobCleanUpPolicy: retain
+                  monitoring: false
+                  annotationCheck: 'false'
+                  engineState: 'active'
+                  chaosServiceAccount: litmus-admin
+                  experiments:
+                    - name: pod-cpu-hog
+                      spec:
+                        probe:
+                        - name: "check-frontend-access-url"
+                          type: "httpProbe"
+                          httpProbe/inputs:
+                            url: "http://front-end.sock-shop.svc.cluster.local:80"
+                            insecureSkipVerify: false
+                            method:
+                              get:
+                                criteria: "=="
+                                responseCode: "200"
+                          mode: "Continuous"
+                          runProperties:
+                            probeTimeout: 2
+                            interval: 1
+                            retry: 2
+                            probePollingInterval: 1
+                        - name: "check-probe-success"
+                          type: "promProbe"
+                          promProbe/inputs:
+                            endpoint: "http://prometheus.monitoring.svc.cluster.local:9090"
+                            query: "sum(rate(request_duration_seconds_count{job='sock-shop/front-end',route='/',status_code='200'}[20s]))*100"
+                            comparator:
+                              criteria: ">=" #supports >=,<=,>,<,==,!= comparision
+                              value: "500"
+                          mode: "Edge"
+                          runProperties:  
+                            probeTimeout: 5
+                            interval: 5
+                            retry: 1
+                        components:
+                          experimentImage: "litmuschaos/go-runner:latest"
+                          env:
+                            - name: TARGET_CONTAINER
+                              value: ''
+
+                            - name: NODE_CPU_CORE
+                              value: '1'
+
+                            - name: TOTAL_CHAOS_DURATION
+                              value: '' # in seconds
+
```

Comment:
why we are not providing `TOTAL_CHAOS_DURATION` here?

Reply:
Nothing as such because it will take  default

Reply:
It is in the engine so it will override the value In the experiment so it should not take the default value.

Reply:
yes @uditgaurav, It will take from the engine. 
That is why I didn't take into the experiment. 
In engine like pod-cpu-hog didn't pass from engine and experiment will have default one. 
Will give value there, Ok?

Reply:
In engine it is empty Oum.

Reply:
PTAL @uditgaurav